### ISSUE TITLE
Which Android SDK packages / openjdk to use? [godot 3.5.3]

### ISSUE THREAD

**Issue by SnailRabbids** [godotengine/godot-docs#8822]
**Your Godot version:**
3.5.3.stable.official

**Issue description:**
I'm trying to update Android SDK and export to Android 13 (target sdk 33). I've found a documentation:
https://docs.godotengine.org/en/3.5/tutorials/export/exporting_for_android.html 
https://docs.godotengine.org/en/stable/tutorials/export/exporting_for_android.html 

Question about Openjdk and packages I need to have. I'm using 3.5.3, so I probably have to use docs for 3.5. But as I want to export to Android 13 (target sdk 33) I need to install everything as in Godot 4.2 docs.

So which docs I have to use?

```
Download and install OpenJDK XX

Ensure that the required packages are installed as well.

- Android SDK Platform-Tools version 30.0.5 or later
- Android SDK Build-Tools version xxx
- Android SDK Platform xxx
- Android SDK Command-line Tools (latest)
- CMake version 3.10.2.4988404
- NDK version r23c (23.2.8568313)
```

**URL to the documentation page (if already existing):**
https://docs.godotengine.org/en/3.5/tutorials/export/exporting_for_android.html 
https://docs.godotengine.org/en/stable/tutorials/export/exporting_for_android.html

**Comment by AThousandShips** [comment]
You should use the 3.x docs for 3.x, not the 4.x docs

**Comment by SnailRabbids** [comment]
@AThousandShips 
So even I need to export for Android 13 (target sdk 33) I have to install "Android SDK Platform 31" ?
See: https://docs.godotengine.org/en/3.5/tutorials/export/exporting_for_android.html#download-the-android-sdk

**Comment by AThousandShips** [comment]
I don't know, I'm not sure what version is supported for 3.x, what SDK

But in either case you should use the 3.x documentation, and if there's something wrong *that* should be updated, but if you're using 3.x you should obviously use the 3.x documentation

**Comment by SnailRabbids** [comment]
I know that I can export for Android 13 (target sdk 33), because I've already tried it. But I don't know if I have to update "Android SDK Platform" from "31" to "33". Its not clear explained in docs, thats why I created this issue.

**Comment by AThousandShips** [comment]
Then the issue should be for the 3.x documentation, not related to the 4.x at all 🙂

**Comment by SnailRabbids** [comment]
I don't see any tags like 3.x or 4.x. How could I move this issue to the 3.x version?

**Comment by AThousandShips** [comment]
There isn't, you just update your issue to focus on that instead of what version of the documentation to use

**Comment by skyace65** [comment]
Fixed by #9105.
